net/dns/manager_tcp_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package dns
```

Comment:
blank line separating the comment and package line, otherwise it becomes a package doc comment